[doc] New data in onChatStarted callback (#157)
iadvize/documentation: docs/javascript-callbacks.md

- Called when : a chat discussion has started.
- Parameter(s) : `obj` contain 2 values:

>   - `obj.id` -> Chat identifier that you can use with our REST API

Reviewer: ```suggestion
  - `obj.id` -> Chat identifier which can be used with our REST API
```

>   - `obj.conversationId` -> Conversation UUID that you can use with our GraphQL API

Reviewer: ```suggestion
  - `obj.conversationId` -> Conversation UUID which can be used with our GraphQL API
```

>   - `obj.vuid` -> Visitor Uniq Id is a random string that you can use for analytics purposes

Reviewer: ```suggestion
  - `obj.vuid` -> Visitor Unique Id is a random string which can be used for analytics purposes
```
